pyproject.toml

```diff
@@ -40,7 +40,7 @@ dependencies = [
   "opentelemetry-instrumentation-aiohttp-client==0.33b0",
   "opentelemetry-instrumentation-asgi==0.33b0",
   "opentelemetry-sdk>=1.9.0",
-  "opentelemetry-semantic-conventions==0.33b0",
+  "opentelemetry-semantic-conventions==0.34b0",
```

Comment:
Why do we only bump `opentelemetry-semantic-conventions`? Should we bump all other opentelemetry dependencies as well?